srcpkgs/signond/template

```diff
@@ -4,7 +4,7 @@ version=8.60
 revision=2
```

Comment:
Please bump revision

Reply:
I'm sorry but I mistakenly deleted the branch.
How can we solve it?

Reply:
https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches

Reply:
I had to recreate a new void-packages clone because i got a strange error therefore i lost all branches reference. I think that the previous branch is not repairable. How to proceed?

Reply:
The above and https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project . Deleting old clone is likely overreaction if git wasn't killed with `^C`, few `checkout` commands should repair a clone.

Reply:
sorry, I tried it but i can't repair.